CONTRIBUTING.md

```diff
@@ -15,31 +13,6 @@ Kubeflow Training Operator is currently at v1.
   - [Colima](https://github.com/abiosoft/colima) (Lima specifically for MacOS) (0.6.8 or later)
 - [pre-commit](https://pre-commit.com/)
 
-Note for Lima the link is to the Adopters, which supports several different container environments.
```

Comment:
I think, it would be useful to explain in contributor guide on how to leverage our Makefile commands to develop Kubeflow Trainer project.
For example:
Run `make generate` to generate code assets.
Run `make test` to run Go unit tests
Run `make test-integration` to run Go integration tests.
etc..